src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -2094,7 +2096,7 @@ private static List<Event> simpleSplitIntoBiallelics(final VariantContext vc) {
         final List<Event> result = new ArrayList<>();
 
         if (vc.isBiallelic()) {
-            return Collections.singletonList(Event.ofWithoutAttributes(vc));
+            return Collections.singletonList(Event.ofWithoutAttributes(GATKVariantContextUtils.trimAlleles(vc, true, true)));
```

Comment:
Did you need this for testing?

Reply:
Nope, this is the fix from #8445. Let me rebase and this should disappear from the diff..